packages/apps/docs/src/pages/docs/pact/index.tsx

```diff
@@ -0,0 +1,186 @@
+import { Button, Card, Grid, Heading, Stack, Text } from '@kadena/react-ui';
+
+import { BrowseSection } from '@/components';
+import { MostPopular } from '@/components/MostPopular';
+import { IMostPopularPage } from '@/types/MostPopularData';
+import getMostPopularPages from '@/utils/getMostPopularPages';
+import {
+  checkSubTreeForActive,
+  getPathName,
+} from '@/utils/staticGeneration/checkSubTreeForActive.mjs';
+import { GetStaticProps } from 'next';
+import Link from 'next/link';
+import React, { FC } from 'react';
+
+interface IProps {
+  popularPages: IMostPopularPage[];
+}
+
+const Home: FC<IProps> = ({ popularPages = [] }) => {
+  return (
+    <Stack direction="column" spacing="$2xl">
+      <Stack direction="column" spacing="$md">
+        <Stack direction="column">
+          <Heading as="h4">Smart-contract language</Heading>
+          <Text as="p">
+            designed for correct, transactional execution on a high-performance
+            blockchain.
+          </Text>
+        </Stack>
+        <Grid.Root columns={10} spacing="md">
+          <Grid.Item
+            columnSpan={{
+              sm: 10,
+              md: 7,
+            }}
+          >
+            <Card>
+              <Stack direction="column" spacing="$sm">
+                <Heading as="h6">Getting Started is Simple</Heading>
+                <Text as="p">
+                  Learn Kadena&apos;s core concepts & tools for development in
+                  15 minutes.
+                </Text>
+                <Stack direction="row" spacing="$sm">
+                  <Button
+                    title="Get Started"
```

Comment:
something a little off in responsive:
![image](https://github.com/kadena-community/kadena.js/assets/4015521/2873dc99-af86-4f3f-869f-07fb5a70c0bd)